scripts/21-import_neighborhood.sh

```diff
@@ -3,30 +3,17 @@ set -euo pipefail
 [ "${PFB_DEBUG}" -eq "1" ] && set -x
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
-# NB_TEMPDIR="test/data"
+IMPORT_SRID=4326
 NB_MAX_TRIP_DISTANCE=2680
 NB_BOUNDARY_BUFFER=$NB_MAX_TRIP_DISTANCE
+NB_COUNTRY=$(echo "$NB_COUNTRY" | tr '[:lower:]' '[:upper:]')
+SHP2PGSQL_CMD="shp2pgsql -d -I -D -s ${IMPORT_SRID}:${NB_OUTPUT_SRID}"
```

Comment:
As per the man page, the reprojection does not work with the `-D` flag, therefore this patch is not valid. 

Better using the original function then.